### ISSUE TITLE
Oswald: interpolation errors in ‘:’ and ‘„’

### ISSUE THREAD

**Issue by fitojb** [google/fonts#658]
: (U+003A) and „ (U+201E) have interpolation errors in Medium, Semibold and Bold.
![captura de pantalla de 2017-02-15 01-26-54](https://cloud.githubusercontent.com/assets/554953/22964376/e9f0e230-f31d-11e6-8fbd-e519af366cc1.png)

**Comment by m4rc1e** [comment]
@fitojb Thanks! I really appreciate your contributions.

I added a test for this in my glyphsapp QA tool a few months back.

![screen shot 2017-02-15 at 09 54 09](https://cloud.githubusercontent.com/assets/7525512/22969019/c2c43e5e-f364-11e6-8c7c-15ef09298bbd.png)

These fonts were updated before we realised Glyphs doesn't flag these MM issues. Glyphs thinks these glyphs are compatible because they're just double component glyphs. It can't workout if the order is correct.
